scapy/layers/can.py

```diff
@@ -9,12 +9,20 @@
 
 """
 
-
+import six
```

Comment:
Hi !
Please use:
`import scapy.modules.six as six`
or
`from scapy.modules import six`

We pack six with scapy so we have no depedencies.

Nice PR otherwise 👍